python/python/Ice/__init__.py

```diff
@@ -35,214 +28,97 @@
 AsyncInvocationContext = IcePy.AsyncInvocationContext
 Unset = IcePy.Unset
 
-from .Future import Future, wrap_future
-from .InvocationFuture import InvocationFuture
-
-from .Object import Object
-from .Blobject import Blobject
-from .BlobjectAsync import BlobjectAsync
-from .Value import Value
-from .FormatType import FormatType
-from .ModuleUtil import *
-
 #
-# This value is used as the default value for struct types in the constructors
-# of user-defined types. It allows us to determine whether the application has
-# supplied a value. (See bug 3676)
+# Forward declarations.
 #
-# TODO: can we use None instead?
-_struct_marker = object()
```

Comment:
Removed, replaced by None

See the mentioned bug https://zeroc.atlassian.net/jira/software/c/projects/ICE/issues/ICE-3381?jql=project%20%3D%20%22ICE%22%20AND%20%22bug%20import%20id%5Bshort%20text%5D%22%20~%20%223676%22%20ORDER%20BY%20created%20DESC